packages/website/components/enterpriseTierRequest/enterpriseTierRequestModal.js

```diff
@@ -0,0 +1,83 @@
+import { useState } from 'react';
+
+import Modal from 'modules/zero/components/modal/modal';
+import CloseIcon from 'assets/icons/close';
+import Button from 'components/button/button.js';
+import { createEnterpriseTierRequest } from 'lib/api';
+import GradientBackground from 'components/gradientbackground/gradientbackground';
+
+const UserRequestModal = ({ isOpen, onClose }) => {
+  const [requesting, setRequesting] = useState(false);
+
+  async function handleCreateUserRequest(e) {
```

Comment:
It will be a bit more performant to wrap this with a `useCallback()` hook. https://reactjs.org/docs/hooks-reference.html#usecallback

Reply:
I'll look into it.  I didn't put too much thought into this as I was just copying from the existing modal.

Reply:
OK, I think I've fixed it, but I want to leave this comment unresolved until you verify it.